app/views/me/notices/index.html.haml

```diff
@@ -1,12 +1,24 @@
-.h2.first=:notices.t
+%div
+  %span.h2.first=:notices.t
+  - if @notices.total_pages < 2 && @notices.any?
+    %span.h2.dissmiss-all-notices-button.dismiss-all-wrapper.pull-right
+      = dissmiss_all_notices_button
+
 - if @notices.empty?
   %ul.shaded
     %li= :none.t
 - else
-  = pagination_links @notices
+  %div
+    .pagination-wrapper.pull-left
+      = pagination_links @notices
+    - if @notices.total_pages > 1
```

Comment:
It is a nice point! I was wondering why everywhere in crabgrass there are
double pagination block, which is uncommon in those days.
I even through that it is a bug, but after found a commit where it was made
intentionally, so decided to leave it as it is :)

It appears in many places, like private messages for example.

Personally don't like duplicated pagination too :)
On Jul 11, 2015 5:34 PM, "azul" <notifications@github.com> wrote:

> In app/views/me/notices/index.html.haml
> <https://github.com/riseuplabs/crabgrass-core/pull/269#discussion_r34413148>
> :
>
> >  - if @notices.empty?
> >    %ul.shaded
> >      %li= :none.t
> >  - else
> > -  = pagination_links @notices
> > +  %div
> > +    .pagination-wrapper.pull-left
> > +      = pagination_links @notices
> > +    - if @notices.total_pages > 1
>
> How about removing the upper pagination block? I don't see why we would
> need it. You scroll down to read all notices and then move on to the next
> page.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/riseuplabs/crabgrass-core/pull/269/files#r34413148>.
>